test/transaction.ts

```diff
@@ -1335,6 +1464,22 @@ describe('Transaction', () => {
         assert.strictEqual(reqOpts.transactionId, id);
       });
 
+      it('should set transactionTag when not single use transaction', done => {
```

Comment:
It should only do so if it is a read/write transaction. Read-only transactions should not contain a transaction tag.

Reply:
This test is in the Transaction sub-module, Snapshot sub-module does not have such a test